assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -293,20 +140,29 @@ def initSentry(sentry_dsn):
 def main():
     args = handle_arguments()
     is_sentry_enabled = initSentry(args["sentry_dsn"])
+
+    config = {
+        "inventory_url": args["assisted_service_url"],
+        "backup_destination": args["backup_destination"],
+        "offline_token": args["offline_token"],
+        "sentry": {
+            "enabled": is_sentry_enabled,
+        },
+        "elasticsearch": {
+            "host": args["es_server"],
+            "index": args["index"],
+            "username": args["es_user"],
+            "password": args["es_pass"],
+        },
+        "max_idle_minutes": int(args["max_idle_minutes"]),
+        "errors_before_restart": int(args["errors_before_restart"]),
+        "n_workers": int(args["n_workers"])
+    }
```

Comment:
I think it is better to create this config as dataclass (autocomplete, type annotations and more)

Reply:
agree